ott/tools/sinkhorn_divergence.py

```diff
@@ -29,6 +34,14 @@ class SinkhornDivergenceOutput(NamedTuple):
                 Optional[jnp.ndarray]]
   converged: Tuple[bool, bool, bool]
 
+  def to_dual_potentials(self) -> "potentials.EntropicMap":
+    """Return the entropic map estimator."""
+    geom_xy, *_ = self.geoms
+    (f_xy, g_xy), (f_x, _), (_, g_y) = self.potentials
+    f, g = f_xy + f_x, g_xy + g_y
```

Comment:
nice! hadn't gone that far :) but it's indeed true that this is simpler in that case.